src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -46,21 +46,26 @@ public Command parseCommand(String userInput) throws ParseException {
         switch (commandWord) {
 
         case AddCommand.COMMAND_WORD:
+        case AddCommand.COMMAND_WORD_ALIAS:
             return new AddCommandParser().parse(arguments);
 
         case EditCommand.COMMAND_WORD:
+        case EditCommand.COMMAND_WORD_ALIAS:
             return new EditCommandParser().parse(arguments);
 
         case DeleteCommand.COMMAND_WORD:
+        case DeleteCommand.COMMAND_WORD_ALIAS:
```

Comment:
Consider adding a comment to more explicitly signal the fall-through
https://se-education.org/guides/conventions/java/index.html
![image](https://user-images.githubusercontent.com/43949290/196093804-fc3dffaf-cdd7-4535-8ab8-2203c2d73f6d.png)